Release v3.5.2

percona-csalguero opened the pull request:
Upgraded to Go 1.20

percona-csalguero committed:
Release v3.5.1 Go 1.20 and upgrade deps

percona-csalguero committed:
Merge branch 'release-v3.5.1' of percona.github.com:percona/percona-toolkit into release-v3.5.2

percona-csalguero committed:
Release v3.5.2

svetasmirnova reviewed (changes requested):
It is too early to mege this into the main branch. We have to update the ChangeLog, release notes, and do all the doc staff first.

alina-derkach-oaza committed:
PT-2193 Release Notes 3.5.2 (#610)

modified:   Changelog
	modified:   Makefile.PL
	modified:   config/sphinx-build/conf.py
	modified:   docs/release_notes.rst
	new file:   docs/rn.3-5-2.txt

svetasmirnova committed:
Updated version properly (#611)

* Release v3.5.1 Go 1.20 and upgrade deps

* Release v3.5.2

* PT-2193 Release Notes 3.5.2 (#610)

modified:   Changelog
	modified:   Makefile.PL
	modified:   config/sphinx-build/conf.py
	modified:   docs/release_notes.rst
	new file:   docs/rn.3-5-2.txt

* Updated version properly

---------

Co-authored-by: Carlos Salguero <carlos.salguero@percona.com>
Co-authored-by: Alina Derkach <alina.derkach@percona.com>
